recipes/hostess/meta.yaml

```
@@ -0,0 +1,59 @@
+{% set name = "hostess" %}
+{% set version = "0.7.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/hostess-{{ version }}.tar.gz
+  sha256: 319916189bc0cb8cf2c85cc2479f5e90cffcc858774a0734856c1365e3cdc912
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: true  # [win]
+  skip: true  # [py<39]
```

This looks like it can be
```suggestion
  noarch: python
```

Any reason not to?

---

We need a python version specification (3.9+) and when adding that while using `noarch: python` the bot replied:
"noarch packages can't have skips with selectors. If the selectors are necessary, please remove noarch: python."

Prior to that I had added the windows skip because otherwise the cicd does not pass as windows is not compatible with this package due to the python-magic dependency (which is not compatible with windows).